ui/src/components/Map/SeverityFloatingFilter.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+      <div>
+            &ge;
+            <select
+                  :style="{ width: '70px' }"
+                  v-model="currentValue"
+                  @change="onSelectionChanged()"
+            >
+                  <option v-for="option in severities" :value="option" :key="option">{{ option }}</option>
+            </select>
+      </div>
+</template>
+
+<script>
+export default {
+      data: function () {
+            return {
+                  severities: [
+                        'Normal',
+                        'Warning',
+                        'Minor',
+                        'Major',
+                        'Critical'
+                  ],
+                  currentValue: ""
+            };
+      },
+      methods: {
+            onSelectionChanged() {
+                  if (this.currentValue === '') {
+                        // clear the filter
+                        this.params.parentFilterInstance((instance) => {
+                              instance.onFloatingFilterChanged(null, null);
+                        });
+                        return;
+                  }
+
+                  this.params.parentFilterInstance((instance) => {
+                        instance.onFloatingFilterChanged('contains', this.currentValue);
+                  });
+            },
+
+            onParentModelChanged(parentModel) {
+                  // When the filter is empty we will receive a null value here
+                  if (!parentModel) {
+                        this.currentValue = '';
+                  } else {
+                        this.currentValue = parentModel.filter;
+                  }
+            }
```

Comment:
Files here generally need formatting to conform to the .prettierrc file rules. This will change the indentation among other things. The rules in that file are just my opinion about what works best so let me know if you disagree with any. Whatever rules we pick though, we'll just need to make sure it's consistent everywhere. 

If you're using vs code, you can use the Prettier formatter extension to comply with the rules. I assume other editors will have similar plugins.